### ISSUE TITLE
Copy-paste (or Duplicate) of MIDI tracks doesn't preserve mute

### ISSUE THREAD

**Issue by Paul-Licameli** [audacity/audacity#2790]
**Describe the bug**

Certain data lost on copy and paste of Label or MIDI tracks

**To Reproduce**
1. Import a MIDI file
2. Mute the MIDI track
3. Select all (Ctrl + A) and copy (Ctrl + C)
6. Pick in the blank below tracks to deselect all
7. Paste
8. New track does not preserve mute status.  This is unlike the behavior for audio tracks.

**Comment by petersampsonaudacity** [comment]
Testing on W10 with 3.1.3 and latest 32.0 alpha master audacity-win-3.2.0-alpha-20220405+793cd28-x64-msvc2022

I confirm that the mute status is **_NOT_** preserved with the Copy & Paste.  

Note that it i**_s_** preserved if you use the Duplicate command after step 3

But Note that in **_both_** cases the copy/duplicate has its vertical scale **_significantly_** changed:
![image](https://user-images.githubusercontent.com/50205984/162575485-9b6b6963-dafa-4a48-97d7-3ecd508995b0.png)

**Comment by Paul-Licameli** [comment]
Make that ruler problem a separate issue

**Comment by petersampsonaudacity** [comment]
> 



> Make that ruler problem a separate issue

ACK

**Comment by petersampsonaudacity** [comment]
Neither are the case if you copy or duplicate a muted audio track

![image](https://user-images.githubusercontent.com/50205984/162575617-0f6c348b-03a4-47cb-adda-40cc4a4da8b9.png)

**Comment by Ryan-Miller-Audacity** [comment]
Tested and confirmed within Windows, MacOS, and Linux via PR #2805. Midi tracks now carry Mute values when copied/pasted and Audio track behaviors remain working as expected.

**Comment by Ryan-Miller-Audacity** [comment]
Unfortunately it does not

**Comment by Paul-Licameli** [comment]
> Unfortunately it does not

I think though that it could be easily fixed now with analogous techniques as in this PR.

A minor and easy fix though beside the priorities of the present release.
